src/Components/gunwookim/SpanLink.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+
+import './SpanLink.scss';
+
+function SpanLink(props) {
+  return (
+    <div className="SpanLink">
+      <Link to={props.to}>
+        <span>{props.text}</span>
```

Comment:
```suggestion
function SpanLink({ text, to }) {
  return (
    <div className="SpanLink">
      <Link to={to}>
        <span>{text}</span>
```
함수 컴포넌트에서 받는 props를 곧바로 구조분해할당 하는 것도 좋아보입니다